.flake8

```diff
@@ -1,2 +1,2 @@
 [flake8]
-max-line-length = 120
\ No newline at end of file
+max-line-length = 220
```

Comment:
220?

Reply:
Can I put infinite? If black doesn't change it, it's fine.